alerts/google-gke/cpu-limit-utilization-containers-within-cluster.v1.json

```diff
@@ -18,7 +18,7 @@
         ],
         "comparison": "COMPARISON_GT",
         "duration": "0s",
-        "filter": "resource.type = \"k8s_container\" AND metric.type = \"kubernetes.io/container/cpu/limit_utilization\" AND resource.labels.cluster_name=\"${CLUSTER_NAME}\" AND resource.labels.location=\"${CLUSTER_LOCATION}\"",
+        "filter": "resource.type = \"k8s_container\" AND metric.type = \"kubernetes.io/container/cpu/limit_utilization\" AND resource.labels.cluster_name=~\"${CLUSTER_NAME}\" AND resource.labels.location=~\"${CLUSTER_LOCATION}\"",
```

Comment:
Revert the =~ back to = for anything that's not PromQL - I think the syntax for ListTimeSeries/Filter alerts is different and it's not worth it to look it up and change these